cache-config/testing/docker/variables.env

```diff
@@ -15,6 +15,10 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+# OS_DISTRO may be set to "centos", default is "rockylinux/rockylinux" since RHEL 8
```

Comment:
`"rockylinux/rockylinux` can be `"rockylinux`

Reply:
@zrhoffman ok, just fixed it.